user-docs/documentation/docs/contracts/api-gateway.md

```diff
@@ -148,32 +148,36 @@ in order to validate the input and/or the output of your lambda.
 
 ### Type the lambda input and output
 
-On the handler side, you can use the `handler` method on the contract to correctly infer the input and output types from the schema.
+On the handler side, you can use the `HandlerType` generic type on the contract to correctly infer the input and output types from the schema.
 
 ```ts
-const handler = myContract.handler(async event => {
+const handler: HandlerType<typeof myContract> = async event => {
   event.pathParameters.userId; // will have type 'string'
```

Comment:
Since we are most likely going to implement parsing and validation in Swarmion, it may be a good idea to keep a wrapper function. It could look something like 
```
const handler = contractHandler(myContract)(async event => {
  event.pathParameters.userId; // will have type 'string'
  ...
});
 ```